onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -1691,6 +1691,8 @@ def test_register_custom_e_ps_library(self):
         # skip amd gpu build
         if "kRocmExecutionProvider" in available_eps:
             return
+        if "kOpenVINOExecutionProvider" in available_eps:
```

Comment:
lines 1692 and 1694 have a problem. it should be 'ROCMExecutionProvider' and 'OpenVINOExecutionProvider'